From foo@baz Tue Apr  9 12:12:43 2002
Date: Thu, 20 Apr 2006 15:50:55 +0900
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: PCI: fix via irq SATA patch

This device id improperly got added to the VIA chipset list with a
previous patch.  Remove it as it is not correct.

Cc: Grzegorz Janoszka <Grzegorz@Janoszka.pl>
Cc: Jeff Garzik <jeff@garzik.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/i386/pci/irq.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/arch/i386/pci/irq.c
+++ gregkh-2.6/arch/i386/pci/irq.c
@@ -591,7 +591,6 @@ static __init int via_router_probe(struc
 	case PCI_DEVICE_ID_VIA_8233A:
 	case PCI_DEVICE_ID_VIA_8235:
 	case PCI_DEVICE_ID_VIA_8237:
-	case PCI_DEVICE_ID_VIA_8237_SATA:
 		/* FIXME: add new ones for 8233/5 */
 		r->name = "VIA";
 		r->get = pirq_via_get;
